test/src/test/java/hudson/cli/QuietDownCommandTest.java

```diff
@@ -129,7 +129,7 @@ public void quietDownShouldFailWithEmptyTimeout() throws Exception {
 
     @Test
     public void quietDownShouldSuccessOnAlreadyQuietDownedJenkins() throws Exception {
-        j.jenkins.getActiveInstance().doQuietDown();
+        j.jenkins.get().doQuietDown();
```

Comment:
Could this  (and other occurences in tests) be changed to `Jenkins.get`?

Reply:
I do not get the question. What are you suggesting?

Reply:
Also not clear to me

Reply:
Always accessing static functions statically (`Jenkins.get()` instead of `j.jenkins.get()`) might improve consistency. (Just an opinion, not a blocker.)

Reply:
Should actually be 
```suggestion
        j.jenkins.doQuietDown();
```

Reply:
Counterpoint to statics: https://stackoverflow.com/questions/7026507/why-are-static-variables-considered-evil

Just reference the `Jenkins` instance when possible like with Daniel's diff.